notebooks/Tutorial8-Time_series.py

```diff
@@ -284,4 +288,85 @@ def plot_series(
+
+# +
+# we get the unshuffled vanilla avatars from the job result based on the original id
+vanilla_unshuffled_avatars = [
+    result.sensitive_unshuffled_avatars_datasets
+    for result in job.result.datasets
+    if result.original_id == dataset_vanilla.id
+][0]
+
+# we get the unshuffled time series avatars from the job result based on their original ids
+ts_unshuffled_avatars = [
+    result.sensitive_unshuffled_avatars_datasets
+    for result in job.result.datasets
+    if result.original_id != dataset_vanilla.id
+]
+ts_original_ids = [
+    result.original_id
+    for result in job.result.datasets
+    if result.original_id != dataset_vanilla.id
+]
+
+# +
+privacy_job = client.jobs.create_privacy_metrics_time_series_job(
+    PrivacyMetricsWithTimeSeriesJobCreate(
+        parameters=PrivacyMetricsWithTimeSeriesParameters(
+            vanilla_original_id=dataset_vanilla.id,
+            vanilla_unshuffled_avatars_id=vanilla_unshuffled_avatars.id,
+            time_series=[
+                PrivacyMetricsTimeSeriesParameters(
+                    dataset_id=original_id,
+                    unshuffled_avatar_dataset_id=avatar.id,
+                    projection_parameters=FPCAParameters(nf=3),
+                )
+                for original_id, avatar in zip(ts_original_ids, ts_unshuffled_avatars)
+            ],
+        )
+    )
+)
+
+privacy_job = client.jobs.get_privacy_metrics_time_series_job(
+    privacy_job.id, timeout=10
+)
+# -
+
+# Metric results are calculated for each dataset and are stored in `privacy_job.result`
+
+for metric_details in privacy_job.result.details:
+    print("-------------")
+    print(f"Metrics for avatar of original dataset {metric_details.original_id} are: ")
+    print(metric_details)
+
+# ## Signal metrics for time series data
+
+# +
+signal_job = client.jobs.create_signal_metrics_time_series_job(
+    SignalMetricsWithTimeSeriesJobCreate(
+        parameters=SignalMetricsWithTimeSeriesParameters(
+            vanilla_original_id=dataset_vanilla.id,
+            vanilla_avatars_id=vanilla_unshuffled_avatars.id,
+            time_series=[
+                SignalMetricsTimeSeriesParameters(
+                    dataset_id=original_id,
+                    avatar_dataset_id=avatar.id,
+                )
+                for original_id, avatar in zip(ts_original_ids, ts_unshuffled_avatars)
+            ],
+        )
+    )
+)
+
+signal_job = client.jobs.get_signal_metrics_time_series_job(signal_job.id, timeout=10)
+# -
+
+# Metric results are calculated for each dataset and are stored in `signal_job.result`
+
+for metric_details in signal_job.result.details:
+    print("-------------")
+    print(f"Metrics for avatar of original dataset {metric_details.original_id} are: ")
+    print(metric_details)
```

Comment:
did we add a documentation about time series metrics (in particular for signal metrics :) )

Reply:
indeed, new sections needed in the public doc (separate PR)